sdk/communication/AzureCommunicationChat/AzureCommunicationChat.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,13 @@
 		88D37B9125D60CCD004F3699 /* PhoneNumberIdentifierModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 88D37B8D25D60CCD004F3699 /* PhoneNumberIdentifierModel.swift */; };
 		88D37B9925D60EDB004F3699 /* AzureCommunicationChatClient.swift in Sources */ = {isa = PBXBuildFile; fileRef = 88D37B9825D60EDB004F3699 /* AzureCommunicationChatClient.swift */; };
 		9766BE9261FCB48436FA6682 /* Pods_AzureCommunicationChatTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = F1FE3013E797A134E8A6C3B5 /* Pods_AzureCommunicationChatTests.framework */; };
+		D10458A725EEA47800B3EB6D /* CommunicationSignallingClient.swift in Sources */ = {isa = PBXBuildFile; fileRef = D10458A125EEA47800B3EB6D /* CommunicationSignallingClient.swift */; };
+		D10458A825EEA47800B3EB6D /* TrouterUtils.swift in Sources */ = {isa = PBXBuildFile; fileRef = D10458A225EEA47800B3EB6D /* TrouterUtils.swift */; };
+		D10458D025EEA54700B3EB6D /* CoreTelephony.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D10458CF25EEA54700B3EB6D /* CoreTelephony.framework */; };
+		D10458D225EEA55400B3EB6D /* libc++.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = D10458D125EEA55400B3EB6D /* libc++.tbd */; };
+		D10458D725EEA57400B3EB6D /* libz.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = D10458D625EEA57400B3EB6D /* libz.tbd */; };
+		D10458DC25EEA57E00B3EB6D /* SystemConfiguration.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D10458DB25EEA57E00B3EB6D /* SystemConfiguration.framework */; };
+		D10458DE25EEA5CB00B3EB6D /* libtrouter-client-ios-x64.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D10458DD25EEA5CB00B3EB6D /* libtrouter-client-ios-x64.a */; };
```

Comment:
@glorialimicrosoft can these also be removed? Your change to remove local trouter has been merged already btw in case this also needs to be removed

Reply:
Good point, these (TrouterUtils.swift, CoreTelephony.framework, libc++.tbd, libz.tbd) were added because the old trouter library needed them. With the new package, I don't think we need to include them anymore. @knvsl, could you try removing these and see if it still works?

Reply:
Also, is there the same issue with iOS that exists with Android between Calling and Chat due to the different version of trouter?

Reply:
I am not sure, we should definitely test it @glorialimicrosoft could you try adding calling to the test app with trouter & see? I assume the worst :D but best to confirm

Reply:
removed the libs